packages/client/src/components/TransactionList.js

```diff
@@ -132,52 +62,53 @@ function TransactionDetails({ safeOwners, transaction, onClose }) {
       </div>
     </>
   );
-}
+};
 
-function TransactionListItem({
+const TransactionListItem = ({
```

Comment:
Can we move to a separate file?

Reply:
this file is ~150 lines of code which is pretty average for our codebase, I think its fine to co-locate these as they are very coupled

Reply:
I remember having a similar discussion before, it is true that those two components are very closely coupled, the problem is average length of file and how closely components are coupled are subjective, not as easy to execute on the simple rule of one component per file. As the codebase grows bigger and more complex, sometimes we need these cookie cutter rules to keep the codebase maintainable. @amyliu6 What's your opinion?

Reply:
I would vote to follow CAST's eslint rules so we have parity across the organization: https://github.com/DapperCollectives/CAST/blob/main/frontend/packages/client/package.json#L80-L85

Reply:
That's perfect! I'll make sure to add this rule to our eslint config